### DISCUSSION TITLE
streams-generator-serial not working

### DISCUSSION THREAD

**Discussion by DharmikEC** [pschatzmann/arduino-audio-tools#371] [Q&A] [answered]
So I think this is the most basic code to check board and library and I cant make it work. I have tried using arduino v1.8.13 and arduino v2.0.0 with ESP32 v2.0.5 and with both IDEs this example code shows no response on serial plotter.  Where the issue could be ?? ESP-32 is working perfectly as it should be so that is not an issue. I am using ESP dev module as board. 

![Screenshot (125)](https://user-images.githubusercontent.com/79980578/191730171-e2875fee-7fc4-4409-b13a-47a5de0a2c15.png)

**Comment by pschatzmann** [comment]
Hmm, I never had any problems in any environment with this!
Check the Serial monitor first to find the error messages.
Change the log level to info or debug to see more details what's going on...

**Reply by DharmikEC** [reply]
Serial monitor also shows nothing with log level Info. I think there is some conflict between libraries but there are just so many and I cant figure out where to start or what to test.

![image](https://user-images.githubusercontent.com/79980578/191731904-53375cf0-0782-4170-a91c-86b47bcf7417.png)

**Comment by pschatzmann** [comment, marked as answer]
A couple of days ago, there was a bug in the copy functionality (for a day or so). But that has been fixed. I am not sure if this is your issue but did you try to update the library?

**Reply by DharmikEC** [reply]
No but I was actually thinking to remove all libraries and start over. So I will do that now.

**Comment by pschatzmann** [comment]
I was just running a test and I can confirm that me it is working as expected.
I never tried it on Arduino 2.0, but I doubt that this is the issue...

**Reply by DharmikEC** [reply]
I have reinstalled all the libraries related to audio. And now it is working.  I have used arduino 2.0 so that is not an issue. Thank you. for quick response.
